hack/arktos-up-scale-out-poc.sh

```diff
@@ -108,10 +108,10 @@ if [ "${CLOUD_PROVIDER}" == "openstack" ]; then
 fi
 
 # set feature gates if enable Pod priority and preemption
+FEATURE_GATES="${FEATURE_GATES_UP_OUT}"
```

Comment:
is there a reason that we need this assignment?

Reply:
This line actually replaces the line 114.

Reply:
HongWei's suggests to use FEATURE_GATES_COMMON_BASE to instead of FEATURE_GATES_UP_OUT to reduce confusing and ambiguity.